rfcs/0006-or-condition.md

```diff
@@ -10,10 +10,18 @@
 This proposal will suggest adding an OR clause to queries that gather the contract to patient mapping in order
 to improve efficiency on both the client and server. 
 
+## Status
+[Status]: #status
+
+* Status: Rejected
```

Comment:
can you add something to this effect in this section:

This RFC was previously approved but after further discussion within the BFD team it has been retroactively rejected because the parameter passing is inappropriate. If this functionality is still required a new approach should be discussed and the RFC updated and re-reviewed.